src/content/basics/giant-swarm-support/index.md

```diff
@@ -0,0 +1,51 @@
+date = "2019-04-16"
+type = "page"
+weight = 20
+categories = ["basics"]
++++
+
+# Giant Swarm Support
+
+Our support service is one of the key differentiators from our competency. We have built a platform which lets our customer spin clusters up as they please. These clusters are then always in the same predictable state and becomes especially valuable once you need to run multiple clusters for different teams, different stages, different configurations or in different locations.
+
+We are alwas at the side of our customer over the whole journey. For us, having a great platform is as important as having someone to nudge when problems occur. Following a brief description of our support process.
+
+## Customer Support
+
+Our customer support relies on close interactions via Slack, to get bi-directional feedback as quickly as possible. This our first contact line to customers, who can ask all kinds of questions, not only in relations to the platform itself, but many more around the Cloud Native journey.
+
+Our support shifts go from 8:00 a.m. to 6 p.m. on weekdays, but often answer questions well out of those times. We have a rotating shift for support across the team, which is why we keep support focussed on those channels with clear internal handovers.
+
+Once there is a problem that the first line support cannot resolve, it is handed over the `ops duty` engineer, again a rotating 24 hour shift, who has a deeper knowledge of the platform.
+
+Additionally, each customer has a dedicated solution engineer as additional support and first fall back in case the person on first line support is overloaded, and for regular sync meetings. On top of that we have an SRE of the week that handles project level work (e.g. installations) and is the fall back for the ops duty engineer. 
+
+## Operation Support
+
+Giant Swarm platform comes with a monitoring and alert system that helps our operation team to maintain our SLAs through all customer clusters. This is the second duty for the ops duty engineer on call every day watching over all alerts coming from all different environments where our customers run their workloads. The operation shift last 24 hours/7 days per week, so we ensure nothing happens even when errors come up during night or weekends.
+
+The monitoring stack observes the underlying infrastructure of the platform, the networking layer, DNS resolution, Kubernetes main components and another set of targets that gives us a complete view of the health of the system.
+
+Today our mean acknowledge average is around two minutes and we usually resolved the incidents in less than two hours. Obviously, it does not mean all the alerts result in a downtime in customer environment as alerts are set to make sure we fix problems before they lead to real incidents.
+
+Additionally, customers have a special email to reach our ops duty engineer 24/7 in case the notice problems that have not been caught by our monitoring.
```

Comment:
```suggestion
Additionally, customers have a special email address to reach our ops duty engineer 24/7.  In case they notice problems that have not been caught by our monitoring.
```